src/server.js

```diff
@@ -95,23 +95,18 @@ function start(opts) {
   paths.mbtiles = path.resolve(paths.root, paths.mbtiles || '');
   paths.pmtiles = path.resolve(paths.root, paths.pmtiles || '');
   paths.icons = path.resolve(paths.root, paths.icons || '');
+  paths.files = path.resolve(paths.root, paths.files || 'public/files');
```

Comment:
Nice, so if the paths root an files don't exists, the deafult publick files get used?

Can you do me a favor and fix a bug? Can you do the same thing to the icons path right above this and set the default to 'public/resources/images' .

Reply:
If the config object icons does not exist, yes. The path existence check itself follows in the next code block.

I added the change for icons.

Reply:
thanks!